### PR TITLE
added support for Amazon Aurora, updated to support Magento 2.1.8

### PR DESCRIPTION
**Pull request by skellish-aws**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -2,3 +2,6 @@
 	path = submodules/quickstart-aws-vpc
 	url = git@github.com:aws-quickstart/quickstart-aws-vpc.git
 	branch = master 
+[submodule "quickstart-linux-bastion"]
+	path = submodules/quickstart-linux-bastion
+	url = https://github.com/aws-quickstart/quickstart-linux-bastion
```
Comment (vsnyc) [inline .gitmodules]: This should specify git ssh url and `branch = master`

File: templates/rdsaurora.template [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+                },
+                "VpcSecurityGroupIds": [
+                    {
+                        "Ref": "RDSSecurityGroup"
+                    }
+                ],
+                "Tags": [
+                    {
+                        "Key": "Name",
+                        "Value": "AuroraDB-YOUR-STACK-REPLACEME"
```
Comment (vsnyc) [inline templates/rdsaurora.template]: Is this expected to be an external parameter?

File: templates/rdsaurora.template [hunk 3]
```diff
@@ -0,0 +1,332 @@
...
+                        },
+                        {
+                            "Ref": "AWS::NoValue"
+                        }
+                    ]
+                },
+                "Tags": [
+                    {
+                        "Key": "Name",
+                        "Value": "AuroraDB-YOUR-STACK-REPLACEME"
```
Comment (vsnyc) [inline templates/rdsaurora.template]: Same question as Line #194. Is this supposed to be external parameter?

File: templates/rdsaurora.template [hunk 4]
```diff
@@ -0,0 +1,332 @@
...
```
Comment (vsnyc) [inline templates/rdsaurora.template]: Same comment as lines: 194, 266

**Comment by vsnyc** [comment]
Hi Scott, this pull request is against the master, it should be against the develop branch. I can close pull request 9 and change the target to develop for this PR.

Santiago has the following two suggestions in PR 9:
* Simplify the choice to just use Aurora unless Aurora is not supported in a particular region. We believe that the benefits of Aurora outweigh any reason to use MySQL RDS for net-new deployments.

Is there a good way to achieve the above? I can think of adding a condition: `UseRDS` that is true for the following four regions (but we'd have to maintain this list of regions that don't have Aurora support): 

 - cn-north-1
 - us-gov-west-1
 - ap-southeast-1
 - sa-east-1

---
I can take up the suggested parameter value change for `QSS3KeyPrefix`
* QSS3KeyPrefix parameter needs the slash at the end as well as the updated properties like description, allowedpattern, etc. All other Sub functions that consume is need to be updated to expect that slash at the end. These are standard S3 Key Prefix naming styles.

**Comment by santiagocardenas** [comment]
@vsnyc I just updated this PR to point to develop branch.

**Pull request closed without merging**